packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardRememberMe.js

```diff
@@ -15,7 +16,7 @@ LoginCardRememberMe.propTypes = {
 };
 
 LoginCardRememberMe.defaultProps = {
-  label: 'Keep me logged in for 30 days',
+  label: '',
```

Comment:
not sure about defaulting to text that is not handled by default. There is nothing here that will expire login after 30 days, correct?

Reply:
I think null is better so this doesn't show up inadvertently.

Reply:
done